distribution/docs/src/main/resources/content/_securityFramework/federated-identity.adoc

```diff
@@ -0,0 +1,65 @@
+:title: Federated Identity
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:order: 10
+:summary: How a user's identity is shared with federated sources during queries
+
+Identity federation is the sharing of a user's identity and attributes among separate systems. In
+{branding}, it is used when performing federated queries - the user's identity is attached to outgoing
+requests, allowing federated sources to run queries as that user. This is critical because {branding}
+must make authorization decisions for all queries, not just those initiated by the local node.
+
+To represent a user's identity when federating, {branding} uses a SAML assertion. This is the format
+used internally to represent all subjects (even unauthenticated subjects). When federating over REST,
+the assertion is simply attached as an HTTP header to the query request, and extracted by the
```

Comment:
should get rid of _simply_ here. Not supposed to use that in technical writing since that can make the reader feel bad. I would also say that it uses the HTTP Authorization header.

Reply:
End the sentence at "request". 
For the next sentence, move the subject to front:

`The Federated Source extracts the SAML assertion from the request.`